android-components/components/lib/crash/build.gradle

```diff
@@ -12,7 +12,6 @@ buildscript {
 
         dependencies {
             classpath "org.mozilla.telemetry:glean-gradle-plugin:${Versions.mozilla_glean}"
-            classpath ComponentsDependencies.plugin_ksp
```

Comment:
Typo or we really do not need the plugin here? I think we have room in `lib-crash` too, so it would be needed, but I don't see CI failing.

Reply:
This is the build script block and we do not need it for the build itself. 
We still have it below in the top-level dependencies block.